webapp/src/pages/home/Home.js

```diff
@@ -116,26 +124,31 @@ class Home extends Component {
                 <h4 className={this.state.applicationStatus == "Submitted" ? "text-success" : "text-danger"}>
                   {this.state.applicationStatus}
                 </h4>
-                
-                {this.state.applicationStatus == "Submitted" && <p>
-                  Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
-                </p>}
+
+                {this.state.applicationStatus == "Submitted" &&
+                <div>
+                  <p>
+                    Thank you! Your application has been received and is under review. You can expect to hear our decision mid-June.
```

Comment:
Lets remove this line - so we just have information about the offer. Alternatively you could add a table with the event, application date and offer status